File: Iniciante/URI 1963.cs
using System; 
 
class URI {
     
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        double A = double.Parse(input[0]);
        double B = double.Parse(input[1]);
        double aumento = (B * 100.0 / A) - 100.0;
        Console.WriteLine("{0:F2}%", aumento);
    }
    
}

File: Iniciante/URI 1963.dart
import 'dart:io';

void main() { 
  List<String> input = stdin.readLineSync().split(" ");
  double A = double.parse(input[0]);
  double B = double.parse(input[1]);
  double aumento = (B * 100.0 / A) - 100.0;
  print("${aumento.toStringAsFixed(2)}%");
}